iree/hal/vulkan/CMakeLists.txt

```diff
@@ -232,6 +233,25 @@ iree_cc_library(
   PUBLIC
 )
 
+if(${CMAKE_HOST_SYSTEM_NAME} STREQUAL "Windows")
```

Comment:
can this be moved to build_tools/third_party/blah? It'd be nice to keep the cmake files we have mostly just declarative usage

Reply:
Yep! I had the same thought but wasn't quite sure where to put it. How's this?